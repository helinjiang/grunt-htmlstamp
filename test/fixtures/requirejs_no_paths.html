<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>requirejs_complex.html</title>
    <script data-main="./requirejs/page/requirejs_complex.js" src="./requirejs/lib/require.js"></script>
</head>
<body>
<h1>requirejs_complex.html</h1>
</body>
</html>