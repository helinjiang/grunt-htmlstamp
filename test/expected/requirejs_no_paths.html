<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>requirejs_no_paths.html</title>
    <script data-main="./requirejs/page/requirejs_no_paths.6ce8ae2e02.js" src="./requirejs/lib/require.js"></script>
</head>
<body>
<h1>requirejs_no_paths.html</h1>
</body>
</html>